app/src/main/java/com/example/diceroller/DiceFragment.kt

```diff
@@ -1,82 +1,73 @@
 package com.example.diceroller
 
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
 import androidx.core.content.ContextCompat
-import androidx.core.graphics.drawable.toDrawable
+import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.lifecycleScope
-import androidx.lifecycle.whenCreated
+import androidx.navigation.fragment.findNavController
 import com.example.diceroller.databinding.FragmentDiceBinding
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.collectLatest
-import kotlinx.coroutines.flow.observeOn
+import kotlinx.coroutines.flow.collect
 
 class DiceFragment : Fragment() {
-    var diceType:MutableStateFlow<DiceType> = MutableStateFlow(DiceType.D6)
-    private var _binding: FragmentDiceBinding? = null
-    private val binding get() = _binding!!
```

Comment:
Oh perfect, I was a little confused reading the documentation because in some places I saw they initiated them like this instead of using lateinit, good to know that they are pretty much the same and at the same time lateinit it's far more clear and readable.